apps/openmw/mwmechanics/spellcasting.cpp

```diff
@@ -433,22 +426,28 @@ namespace MWMechanics
                                     "meshes\\" + reflectStatic->mModel, ESM::MagicEffect::Reflect, false, "");
                         reflectedEffects.mList.push_back(*effectIt);
                         magnitudeMult = 0;
+                        break;
```

Comment:
Shouldn't this read ```continue;```? If an effect is reflected, the rest of the effects list is not evaluated, which seems wrong. The ```if (absorbed)``` statement seems to have the same issue, although that was already there before the PR.